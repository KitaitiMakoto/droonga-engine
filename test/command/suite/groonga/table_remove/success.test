#@include fixture/user-table.jsons
{
  "type": "table_remove",
  "dataset": "Droonga",
  "body": {
    "name"  : "User"
  }
}
