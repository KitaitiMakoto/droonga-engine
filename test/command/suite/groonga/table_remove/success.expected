[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 200,
    "type": "table_remove.result",
    "body": [
      [
        0,
        0.0,
        0.0
      ],
      true
    ]
  }
]
